README.md

```diff
@@ -0,0 +1,3 @@
+[![Build Status](https://travis-ci.org/UCSUR-Pitt/wprdc-etl.svg?branch=master)](https://travis-ci.org/UCSUR-Pitt/wprdc-etl)
+#WPRDC-ETL
+ETL Processes for the Western Pennsylvania Regional DataCenter (www.wprdc.org).
```

Comment:
It would be good to expand this out a bit. Here are some resources to get started with thinking about how to expand this and why it is important:

+ [Readme-driven development](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html)
+ [18F guide to Making READMEs readable](https://pages.18f.gov/open-source-guide/making-readmes-readable/)
+ [Code for America example good README](https://github.com/codeforamerica/howto/blob/master/Good-READMEs.md)